<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addMark</class>
 <widget class="QDialog" name="addMark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="Reper">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>631</width>
     <height>541</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cod</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Denumire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantitate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Greutate/buc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Forma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dimensiuni</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Material</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numar operatii</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="codeLe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="shapeLe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dimensionLe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="materialLe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveMarkBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Salveaza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="priceBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pret</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="operationNameLe">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addOperation">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Adaugare operatie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nume operatie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timp operatie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pret operatie</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="qtySb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="weightSb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="nrOperationSb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="operationPriceSb">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="operationTimeSb">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewMark">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Adauga reper nou</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>       $</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceLe">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="orderNumber">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numar comanda</string>
    </property>
   </widget>
   <zorder>priceLe</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>codeLe</zorder>
   <zorder>nameLe</zorder>
   <zorder>shapeLe</zorder>
   <zorder>dimensionLe</zorder>
   <zorder>materialLe</zorder>
   <zorder>saveMarkBtn</zorder>
   <zorder>priceBtn</zorder>
   <zorder>operationNameLe</zorder>
   <zorder>addOperation</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>qtySb</zorder>
   <zorder>weightSb</zorder>
   <zorder>nrOperationSb</zorder>
   <zorder>operationPriceSb</zorder>
   <zorder>operationTimeSb</zorder>
   <zorder>addNewMark</zorder>
   <zorder>label_12</zorder>
   <zorder>orderNumber</zorder>
   <zorder>label_13</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
